openquake/commands/mosaic.py

```diff
@@ -105,7 +107,9 @@ def from_file(fname, concurrent_jobs=8):
                 continue
             dic = dict(siteid=siteid, lon=float(lon), lat=float(lat))
             tags.append(siteid)
-            allparams.append(get_params_from(dic))
+            # FIXME: should we exclude USA here or not?
```

Comment:
The CSV file does not contain sites in USA (was built so) so there is not need to exclude the USA.